src/_help/specification-support/openapi-support.md

```diff
@@ -5,6 +5,19 @@ title: OpenAPI support
 - TOC
 {:toc}
 
+We support all major versions from Swagger (OpenAPI v2), OpenAPI v3 and OpenAPI v3.1. This pages describes some specificities related to our support for this specification.
+
+## Minimal mandatory fields
+
+Bump.sh needs to receive at least these fields to generate a REST documentation
```

Comment:
```suggestion
Bump.sh needs to receive at least these fields to generate a REST documentation:
```

Reply:
Not sure the punctuation helps that much as it's a short sentence without a full-top